HunterPie/Address/MonsterHunterRise.12.0.0.0.map

```diff
@@ -1,23 +1,23 @@
 ﻿# Addresses
-Address STAGE_ADDRESS 0x0F126578
-Address MONSTERS_ADDRESS 0x0F127EB0
-Address LOCKON_ADDRESS 0x0F131D08
-Address CHARACTER_ADDRESS 0x0F12A8A0
-Address SAVE_ADDRESS 0x0F1197E8
-Address WEAPON_ADDRESS 0x0F124B98
-Address ABNORMALITIES_ADDRESS 0x0F126628
-Address ARGOSY_ADDRESS 0x0F12D500
-Address TRAINING_DOJO_ADDRESS 0x0F12AA80
-Address CHAT_ADDRESS 0x0F11C7C0
-Address CHAT_UI_ADDRESS 0x0F128F80
-Address SESSION_PLAYERS_ADDRESS 0x0F123628
-Address UI_ADDRESS 0x0F128F80
-Address MOUSE_ADDRESS 0x0F127BA0
-Address QUEST_ADDRESS 0x0F130278
-Address MEOWMASTERS_ADDRESS 0x0F1268E8
-Address COHOOT_ADDRESS 0x0F113E50
+Address STAGE_ADDRESS 0x0F126578  <!-- snow.wwise.WwiseChangeSpaceWatcher -->
+Address MONSTERS_ADDRESS 0x0F127EB0  <!-- snow.enemy.EnemyManager -->
+Address LOCKON_ADDRESS 0x0F131D08  <!-- snow.CameraManager -->
+Address CHARACTER_ADDRESS 0x0F12A8A0  <!-- snow.LobbyManager -->
+Address SAVE_ADDRESS 0x0F1197E8  <!-- snow.SnowSaveService -->
+Address WEAPON_ADDRESS 0x0F124B98  <!-- snow.StmInputConfig -->
+Address ABNORMALITIES_ADDRESS 0x0F126628  <!-- snow.envCreature.EnvironmentCreatureManager -->
+Address ARGOSY_ADDRESS 0x0F12D500  <!-- snow.facility.TradeCenterFacility -->
+Address TRAINING_DOJO_ADDRESS 0x0F12AA80  <!-- snow.facility.OtomoDojoFacility -->
+Address CHAT_ADDRESS 0x0F11C7C0  <!-- snow.gui.ChatManager -->
+Address CHAT_UI_ADDRESS 0x0F128F80  <!-- snow.gui.GuiManager -->
+Address SESSION_PLAYERS_ADDRESS 0x0F123628  <!-- snow.player.PlayerManager -->
+Address UI_ADDRESS 0x0F128F80  <!-- snow.gui.GuiManager -->
+Address MOUSE_ADDRESS 0x0F127BA0  <!-- snow.StmInputManager -->
+Address QUEST_ADDRESS 0x0F130278  <!-- snow.QuestManager -->
+Address MEOWMASTERS_ADDRESS 0x0F1268E8  <!-- snow.data.OtomoSpyUnitManager -->
+Address COHOOT_ADDRESS 0x0F113E50  <!-- snow.progress.ProgressOwlNestManager -->
 # Static Data
-Address DATA_TRAINING_DOJO_ROUNDS_LEFT 0x0F12BE98
+Address DATA_TRAINING_DOJO_ROUNDS_LEFT 0x0F12BE98  <!-- snow.facility.OtomoDojoFacility._ChargeNum -->
```

Comment:
Comments for HunterPie's `.map` files start with `#`, having `<!--` in this file will break in the parse process

Reply:
I changed with #.
review please.

Reply:
Did you build it to see if HunterPie loaded the map successfully? I actually don't remember if HunterPie's map parser can parse `#something` instead of `# something`

Reply:
I didn't try.
If you don't mind, i'll check it later.

Reply:
Failed parse, so I edit formatting.
What do you think to modifying the other group as well?